blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -27,6 +27,11 @@ export class Pipelines extends Component {
         this._countExtensions();
     }
 
+    componentDidMount() {
+        document.getElementsByClassName('fastsearch-input')[0].focus();
+        document.addEventListener('keydown', this.handleKeyDownEvent);
```

Comment:
Should always remove listeners / timeouts / intervals in `componentWillUnmount` to avoid memory leaks.